views/login.ejs

```diff
@@ -12,25 +12,25 @@
     <span>Sign in to a test account using OAuth2</span>
 
     <div id="login-form">
```

Comment:
Better iterate through `data.testUsers` in EJS, see for example https://stackoverflow.com/a/31780087/261210

The code should still work for two (or 4) users (4 might break the CSS, but we don't care about that).

Reply:
@nuest Fixed. The css works for 2 users, with 4 or more the last users aren't visible:

![image](https://user-images.githubusercontent.com/5795523/36531606-c4c52b62-17be-11e8-8d40-4989f5edb1ac.png)

Reply:
I fiddled a bit with the CSS and now it's more flexible: https://codepen.io/anon/pen/gvxVjX

![image](https://user-images.githubusercontent.com/1325054/36537893-cc95f318-17d1-11e8-9bc8-1e8df9727f24.png)